core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1031,11 +1029,11 @@ class KafkaApis(val requestChannel: RequestChannel,
         JoinGroupResponse.UNKNOWN_PROTOCOL,
         JoinGroupResponse.UNKNOWN_MEMBER_ID, // memberId
         JoinGroupResponse.UNKNOWN_MEMBER_ID, // leaderId
-        Map.empty[String, ByteBuffer])
+        Map.empty[String, ByteBuffer].asJava)
```

Comment:
It seems like you used `Collections.empty*` methods in a few places. Should we use `emptyMap` here then?